backend/globaleaks/handlers/base.py

```diff
@@ -265,16 +271,19 @@ def validate_request(request, request_template):
 
                 if not BaseHandler.validate_type(request[key], value):
                     log.err("Expected key: %s type validation failure", key)
-                    raise errors.InputValidationError("Key (%s) double validation failure" % key)
+                    raise errors.InputValidationError(
+                        "Key (%s) double validation failure" % key)
 
                 if isinstance(request_template[key], (dict, list)) and request_template[key]:
-                    BaseHandler.validate_request(request[key], request_template[key])
+                    BaseHandler.validate_request(
```

Comment:
@joeman65 @alessiofranceschini: This seems to be just stilistic  code review, please provide this separately to not misk code review of our code with the features you need to implement.